src/argsparser.c

```diff
@@ -301,6 +302,20 @@ void parse_recurring_delay(char *arg)
     free(x);
 }
 
+int isnumber(char *pin)
```

Comment:
That's also the reason why I decided to make a new option at first - to be able specifying just any string without handling possible different situations.